CONTRIBUTING.md

```diff
@@ -87,7 +87,6 @@ This Wiki is intended to be for _anyone_ -- regardless of expertise -- who is in
 
 - Fork this repository in Github's web interface by pressing the fork icon.
 - Clone your fork and `cd` into the new directory: `git clone https://github.com/username/wiki && cd wiki`
-- Check out the "dev" branch: `git checkout -b dev`
 - Make changes as desired to the files in the repository on your machine.
 - Add the changed files to your commit and commit the changes to your fork: `git add file1.md file2.md && git commit -m 'updated x,y,z'`
 - Push your commit to your fork: `git push origin dev`
```

Comment:
- Push your commit to your fork: ` git push origin ha-presence-detection-with-monitor`
- Generate a pull-request using GitHub's web interface.

Reply:
im still learning git/github but im not talking about the github interface here unless im missing something